Compiler/types.py

```diff
@@ -2354,6 +2354,10 @@ def __init__(self, rows, columns, value_type, address=None):
             value_type = _types[value_type]
         self.value_type = value_type
         self.address = Array(rows * columns, value_type, address).address
+        
+    def delete(self):
```

Comment:
Since this uses Array internally, I would have it call the respective function there.

Reply:
`Matrix` does not inherit from `Array`. I am not sure, what is the best practice here.